client/flutter/lib/pages/settings_page.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:WHOFlutter/constants.dart';
+import 'package:WHOFlutter/generated/l10n.dart';
+import 'package:WHOFlutter/components/list_of_items.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+/// TODO: Get all the strings from generated file
+
+class SettingsPage extends StatefulWidget {
+  @override
+  _SettingsPageState createState() => _SettingsPageState();
+}
+
+class _SettingsPageState extends State<SettingsPage> {
+  bool _switchValue = true;
+  int indexSelected = 0;
+
+  @override
+  Widget build(BuildContext context) {
+    return ListOfItems([
+      Padding(
+        padding: const EdgeInsets.symmetric(vertical: 15.0, horizontal: 8.0),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            ListTile(
+              leading: SelectableText(
+                S.of(context).homePagePageSliverListSettingsHeader1,
+                textAlign: TextAlign.start,
+                style: TextStyle(fontSize: 25.0, fontWeight: FontWeight.w700,),
+              ),
+              trailing: CupertinoSwitch(
+                activeColor: Constants.primaryColor,
+                value: _switchValue,
+                onChanged: (value){
+                  /// TODO: Take actions here!
+                  setState(() {
+                    _switchValue = value;
```

Comment:
there are 2 TODOs here - 1 for updating the stored user preferences, and another for actually turning Google Analytics on/off. I think the latter (turning GA on/off) is out of scope of this PR. Storing in user preferences _could_ be a part of this PR though (same comment on line 15). If we're not including it here, make sure to document both TODOs here and make sure that we are tracking this on GitHub issues

Reply:
IMO analytics folks should handle that.  there's some complexity because it is async, so in which order do you do it to guarantee correctness...

Reply:
@advayDev1 Sure thing, I will add detailed todos for this PR and create issues if no one has done it yet.